index.html

```diff
@@ -388,6 +388,28 @@
          as specified in [[!RFC7231]] <a href='https://tools.ietf.org/html/rfc7231#section-4.3.2'>section 4.3.2</a>.
         </p>
       </section>
+
+      <section id="httpDELETE">
+        <h2>HTTP DELETE</h2>
+        <p>
+         The <code>DELETE</code> method is optional per [[!LDP]] <a href='https://www.w3.org/TR/ldp/#ldpr-HTTP_DELETE'>section
+         4.2.5</a> and this specification does not require Fedora servers to implement it.  When a Fedora server supports this method,
+         in addition to the requirements imposed on <a>LDPR</a>s within containers outlined in [[!LDP]]
+         <a href='https://www.w3.org/TR/ldp/#ldpc-HTTP_DELETE'> section 5.2.5</a>, it must also follow the additional behavior outlined
+         below.
+        </p>
+        <section id="httpDELETEDepth">
+          <h3>Depth Header</h3>
+          <p>
+          A Feodra server MAY support the <code>Depth</code> header, as defined in [[!RFC4918]] <a href='https://tools.ietf.org/html/rfc4918#section-10.2'> section
+          10.2 </a>. Implementations MAY choose not to support all of the header's values and MUST reject a request containing an unsupported <code>Depth</code>
+          value with a 400 (Bad Request).  Implementations MUST define their own default behavior if a Depth header is not present.
+          </p>
+          <p>
+          The recursion of the <code>DELETE</code> MUST be reckoned along the <code>ldp:contains</code> axis linking contained resources.
```

Comment:
@awoods Neither of the sources you cite actually supports your understanding. Neither offers any restrictions on what predicates can be used in membership triples.

More to the real point, irrespective of membership triples, non-Basic containers also include containment triples, and those use the `ldp:contains` predicate by definition. Those links would have to be traversed during a `DELETE` by the language you have asked for. That is what we are trying to avoid. This section of the specification should be restricted to action on Basic containers.